desc/compute/_neoclassical_1D.py

```diff
@@ -0,0 +1,461 @@
+    dim=1,
+    params=[],
+    transforms={"grid": []},
+    profiles=[],
+    coordinates="r",
+    data=[
+        "min_tz |B|",
+        "max_tz |B|",
+        "kappa_g",
+        "R0",
+        "|grad(rho)|",
+        "<|grad(rho)|>",
+        "fieldline length",
+    ]
+    + Bounce1D.required_names,
+    resolution_requirement="z",
+    source_grid_requirement={"coordinates": "raz", "is_meshgrid": True},
+    **_bounce1D_doc,
+)
+@partial(jit, static_argnames=["num_well", "num_quad", "num_pitch", "batch"])
+def _epsilon_32_1D(params, transforms, profiles, data, **kwargs):
+    """https://doi.org/10.1063/1.873749.
+
+    Evaluation of 1/ν neoclassical transport in stellarators.
+    V. V. Nemov, S. V. Kasilov, W. Kernbichler, M. F. Heyn.
+    Phys. Plasmas 1 December 1999; 6 (12): 4622–4632.
+    """
+    # noqa: unused dependency
+    num_well = kwargs.get("num_well", None)
+    num_pitch = kwargs.get("num_pitch", 50)
+    batch = kwargs.get("batch", True)
+    if "quad" in kwargs:
+        quad = kwargs["quad"]
+    else:
+        quad = chebgauss2(kwargs.get("num_quad", 32))
+
+    def eps_32(data):
+        """(∂ψ/∂ρ)⁻² B₀⁻² ∫ dλ λ⁻² ∑ⱼ Hⱼ²/Iⱼ."""
+        # B₀ has units of λ⁻¹.
+        # Nemov's ∑ⱼ Hⱼ²/Iⱼ = (∂ψ/∂ρ)² (λB₀)³ ``(H**2 / I).sum(axis=-1)``.
+        # (λB₀)³ d(λB₀)⁻¹ = B₀² λ³ d(λ⁻¹) = -B₀² λ dλ.
+        bounce = Bounce1D(grid, data, quad, automorphism=None, is_reshaped=True)
+        H, I = bounce.integrate(
+            [_dH, _dI],
+            data["pitch_inv"],
+            data,
+            "|grad(rho)|*kappa_g",
+            bounce.points(data["pitch_inv"], num_well=num_well),
+            batch=batch,
+        )
+        return jnp.sum(
+            safediv(H**2, I).sum(axis=-1)
+            * data["pitch_inv weight"]
+            / data["pitch_inv"] ** 3,
+            axis=-1,
+        )
+
+    grid = transforms["grid"].source_grid
+    B0 = data["max_tz |B|"]
+    data["deprecated(effective ripple 3/2)"] = (
+        _compute(
+            eps_32,
+            fun_data={"|grad(rho)|*kappa_g": data["|grad(rho)|"] * data["kappa_g"]},
+            data=data,
+            grid=grid,
+            num_pitch=num_pitch,
+            simp=True,
+        )
+        / data["fieldline length"]
+        * (B0 * data["R0"] / data["<|grad(rho)|>"]) ** 2
+        * jnp.pi
+        / (8 * 2**0.5)
+    )
+    return data
+
+
+@register_compute_fun(
+    name="deprecated(effective ripple)",
+    label="\\epsilon_{\\mathrm{eff}}",
+    units="~",
+    units_long="None",
+    description="Effective ripple modulation amplitude.",
+    dim=1,
+    params=[],
+    transforms={},
+    profiles=[],
+    coordinates="r",
+    data=["deprecated(effective ripple 3/2)"],
+)
+def _effective_ripple_1D(params, transforms, profiles, data, **kwargs):
+    data["deprecated(effective ripple)"] = data["deprecated(effective ripple 3/2)"] ** (
+        2 / 3
+    )
+    return data
+
+
+@register_compute_fun(
+    name="deprecated(Gamma_c)",
```

Comment:
They are not tested in `test_compute_everything`.

I don't want them to show up in the list of variables either.

Reply:
Oh I see, you remove them from the name to compute in that test.

If we can also remove these from docs, that would be great, possibly by adding something to avoid things with `deprecated` in their names to https://github.com/PlasmaControl/DESC/blob/master/docs/write_variables.py